libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -325,6 +325,16 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
                     className="gd-header-menu-horizontal"
                 />
 
+                {this.props.isDisplayUpsellButton && (
+                    <button
```

Comment:
Should we create a new component called, for example, `UpsellButton` and put codes from lines 329~335 to that component?

Reply:
I created new component thanks